server/routerlicious/packages/lambdas/src/utils/validateDocument.ts

```diff
@@ -36,5 +38,13 @@ export function isDocumentSessionValid(
 		// Prevent Deli from processing ops.
 		return false;
 	}
+
+	// If the orderer URL contains the tenant ID, the session is private link enabled.
+	// So far we only set one cluster per group for private endpoint.
+	// Skip the isSessionInThisCluster check.
+	const ordererUrlContainsTenantId = document.session.ordererUrl.includes(document.tenantId);
+	if (ordererUrlContainsTenantId) {
+		return true;
+	}
```

Comment:
isnt this equivalent to: return document.session.ordererUrl.includes(document.tenantId);